src/avram/migrator/runner.cr

```diff
@@ -172,10 +172,25 @@ class Avram::Migrator::Runner
 
   def ensure_migrated!
     if !pending_migrations.empty?
-      raise "There are pending migrations. Please run lucky db.migrate"
+      puts ""
+      puts error_background
+      puts pending_migrations_error.colorize.on_red.white
+      puts error_background
+      puts ""
+      Process.signal(:term, Process.ppid)
+      Process.signal(:hup, Process.ppid)
+      exit 1
```

Comment:
This tells the process manager to shut down, then exit the program with an error code 1. If the parent process doesn't respond to the `term` and `hup` signals, it will call the exit, but then hang. However, you'll still see the bright red banner which is still better than it currently is.

Reply:
The idea here is that we send both term and hup, so that one of those might make a difference? Do we know that both are needed? I would expect term to be sufficient.

Reply:
I actually got the suggestion from someone else, I can try just term and see if that's sufficient though.

Reply:
ok, looks like term is enough. It seems `TERM` and `HUP` are pretty much the same in this scenario since no one would be booting an app in a screen then trying to close their terminal 😂